content/prerequisites/lab-resources/index.md

```diff
@@ -1,20 +0,0 @@
----
```

Comment:
Have we checked if this page is referenced in other labs?  If we are refactoring this page in another location, can we make sure updated links, if any, are set in place?

Reply:
done